### ISSUE TITLE
methodology to follow on Linux

### ISSUE THREAD

**Issue by Hyphaed** [artur-graniszewski/DLSS-Enabler#3]
how should I use this in Linux exactly? lets say in Wukong BlackMyth?

since I see you release the exe for Windows, and the sources

**Comment by artur-graniszewski** [comment]
Hi..

Here's the full mod description:
https://www.nexusmods.com/site/mods/757

you can try to use the windows installer provided by that mod, or.... a Linux installer
https://www.nexusmods.com/site/mods/927

Remember to modify WINEEXPORTS variable accordingly (depending on the way you installed the mod)

**Comment by slynobody** [comment]
i would not use the linux installer linked above in its current state. at first glance its usage is rather simple, but it is far from stable (incl. unforeseeable unintended deletion of files). i currently extract the files from the windows installer and place them in the right context (but running a suggestion for heroic launcher to automate this, esp. for registering the nvidia-reg in each corresponding wine-prefix).

**Comment by artur-graniszewski** [comment]
Thanks for the feedback. Just one question from my side then: did the mod work correctly for you? I'm asking, as manual installation can be challenging due to the amount of files combinations that differ depending on gpu vendor

**Comment by Hyphaed** [comment]
I've tested in Windows + Linux

I do have an RTX 3070

I've tried Wukong Benchmark Tool

I choose DLSS, yet it do not let me enable the Frame Generation Option :: It tells me the GPU is not compatible

Isn't supposed to solve this issue this patch?

**Comment by artur-graniszewski** [comment]
Yes. It should and has been succesfully tested under linux and windows. You most probably installed the mod into wrong directory. It should be installed into b1/Binaries/Win64 one

**Comment by artur-graniszewski** [comment]
In case of this app, version.dll installation option should be used.

**Comment by jeevithakannan2** [comment]
I have similar issue.

Not working in Black Myth: Wukong Benchmark Tool. Launched the game using the steam method. Still game didn't allow me to use dlss frame generation. 

OS: ArchLinux
Kernel: 6.10.6-zen1-1-zen
Proton Version: Proton 9.0-2
GPU: Nvidia Geforce RTX 3050 Ti Mobile
CPU: Intel 12th gen i7 12700H

![240823T022310-screenshot](https://github.com/user-attachments/assets/be5a7057-098a-458f-a7cf-da534073adf4)

**Comment by artur-graniszewski** [comment]
Hi.

Please use latest beta builds (ie 3.01.001.0 beta 11). Linux support for this app has been implemented in one of said betas. 

Everything should work out of the box after installation.

**Comment by artur-graniszewski** [comment]
https://github.com/artur-graniszewski/DLSS-Enabler/releases/tag/3.01.001.0-beta11
